### PR TITLE
Add CSS

### PR DESCRIPTION
**Pull request by Ranapriyanshi**

### CODE REVIEW

File: ornate_frontend/src/index.js [hunk 1]
```diff
@@ -2,16 +2,18 @@ import React from "react";
...
+import Creator from "./pages/Creator";
 const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
   <BrowserRouter>
     <Routes>
-      <Route path="/" element={<Home />}>
-        <Route path="navbar" element={<Navbar />} />
-      </Route>
+      <Route path="/" element={<Home />}></Route>
+      <Route path="/SignUp" element={<SignUp />}></Route>
```
Comment (aditya-190) [inline ornate_frontend/src/index.js]: According to React naming convection routes are generally in `snake_case`. Please try to change them in your later PR's. To know more about, check this out -> https://www.upbeatcode.com/react/react-naming-conventions/

File: ornate_frontend/src/index.js [hunk 2]
```diff
@@ -2,16 +2,18 @@ import React from "react";
...
+import SignUp from "./pages/SignUp";
+import Creator from "./pages/Creator";
 const root = ReactDOM.createRoot(document.getElementById("root"));
 root.render(
   <BrowserRouter>
     <Routes>
-      <Route path="/" element={<Home />}>
-        <Route path="navbar" element={<Navbar />} />
-      </Route>
+      <Route path="/" element={<Home />}></Route>
```
Comment (aditya-190) [inline ornate_frontend/src/index.js]: `Route` is an self closing tag. You can replace it with - `<Route path="/" element={<Home />} />`

File: ornate_frontend/src/pages/Home.js [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+              <div id="save_section">
+                <button className="save_button">
+                  Save <CiSaveDown2 />
+                </button>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div className="cards">
```
Comment (aditya-190) [inline ornate_frontend/src/pages/Home.js]: Small Code optimization can be achieved by extracting the repeated code into a new component and implementing loops to efficiently execute it as many times as required.

**Review by aditya-190 — CHANGES_REQUESTED** [review]
Please make the required changes before merging the changes.

**Review by PatelNeelMahesh — APPROVED** [review]
Since, you completed all the changes you can merge pull request.

**Pull request merged**